packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -198,11 +199,12 @@ function ProductDetails({
                   buyButtonIcon={buyButtonIcon}
                 />
               ) : (
-                <OutOfStock />
+                // TODO: Adds <OutOfStock /> when component is ready to use
+                <p>Not Available</p>
```

Comment:
Is it possible to style this text (via css)?

Reply:
ehr.. 

`data-fs-product-details-section > p {}`

Maybe we want a data-attr for this text 🤔

Reply:
Considering that it is a temporary label and we are not adding any particular style, I'll leave it without data-attr for now, ok?